src/ert/_c_wrappers/analysis/analysis_module.py

```diff
@@ -132,7 +131,9 @@ def variable_value_dict(self) -> Dict[str, Union[float, int]]:
     def handle_special_key_set(self, var_name, value):
         if var_name in self.DEPRECATED_KEYS:
             logger.warning(
-                f"The {var_name} key have been removed" f"use the INVERSION key instead"
+                f"The {var_name} key have been removed"
+                f"use the INVERSION key "
+                f"instead"
```

Comment:
Can have less line breaks, and drop the f-prefix where not needed

Reply:
Reverted back to original state, I think this was done by a save action